src/lightning/data/streaming/downloader.py

```diff
@@ -37,22 +37,25 @@ def download_file(self, remote_chunkpath: str, local_chunkpath: str) -> None:
 
 
 class S3Downloader(Downloader):
+    _client = None
+
     @classmethod
```

Comment:
Why is this a class method? Everywhere we use the downloader, it is an instance. Let's make it an regular method, have the `S3Downloader.__init__()` create an attribute for the client, and reference the client through the object?

Reply:
Sure.